packages/@sanity/schema/src/sanity/validation/types/array.ts

```diff
@@ -1,8 +1,35 @@
+import {flatten, partition} from 'lodash'
+import humanizeList from 'humanize-list'
 import {error, HELP_IDS, warning} from '../createValidationResult'
-import {flatten} from 'lodash'
 import {getDupes} from '../utils/getDupes'
 import {coreTypeNames} from '../../coreTypes'
 
+function isPrimitiveTypeName(typeName) {
```

Comment:
I thought v3 didn't allow implicit any?

Would love to see type info for all params, even if just inline definitions.

Reply:
Good point - seems strict mode has been disabled for this package https://github.com/sanity-io/sanity/blob/2778e0ca004ddcf0e32fb8f4cf91b7323e640b1e/packages/@sanity/schema/tsconfig.json#L10